openlibrary/macros/RawQueryCarousel.html

```diff
@@ -21,13 +21,14 @@
 
 $code:
   # Limit to just fields needed to render carousels
-  params = { 'q': query, 'fields': 'key,title,subtitle,author_name,cover_i,ia,availability,id_project_gutenberg,id_project_runeberg,id_librivox,id_standard_ebooks,id_openstax' }
+  fields = ['key', 'title', 'subtitle', 'editions', 'author_name', 'availability', 'cover_i', 'ia', 'id_project_gutenberg', 'id_librivox', 'id_standard_ebooks', 'id_openstax', 'providers']
```

Comment:
Another issue here, these were previously forwarded to search.json . But I think we can just remove them here, and add `editions` to https://github.com/internetarchive/openlibrary/blob/07adc7b57127f5f2313f15ce196dc894ccf55c5a/openlibrary/plugins/openlibrary/partials.py#L84